RubeesEat/InMemoryStartUpForTests.cs

```diff
@@ -0,0 +1,99 @@
+using System.Collections.Immutable;
+using RubeesEat.Model;
+using RubeesEat.Model.InMemory;
+
+namespace RubeesEat;
+
+public class InMemoryStartUpForTests
```

Comment:
As discussed, we should not need this but instead override services if necessary in the tests.